modules/dnn/test/test_backends.cpp

```diff
@@ -308,6 +308,12 @@ TEST_P(DNNTestNetwork, FastNeuralStyle_eccv16)
         (backend == DNN_BACKEND_OPENCV && target == DNN_TARGET_OPENCL_FP16) ||
         (backend == DNN_BACKEND_INFERENCE_ENGINE && target == DNN_TARGET_MYRIAD))
```

Comment:
please also check if `backend == DNN_BACKEND_OPENCV && target == DNN_TARGET_OPENCL_FP16` and `backend == DNN_BACKEND_INFERENCE_ENGINE && target == DNN_TARGET_MYRIAD` configurations pass this test.